tests/DS/sds_detect_version/test_detect_version.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2019 Red Hat Inc., Durham, North Carolina.
+# All Rights Reserved.
+#
+# OpenSCAP Test Suite
+#
+# Authors:
+#      Jan Černý <jcerny@redhat.com>
+
+. $builddir/tests/test_common.sh
+
+set -e -o pipefail
+
+function test_oscap_info {
```

Comment:
What about change the test to detect which sds are available in the project and test all of them? The Datastream can easily be created by copying and changing the version.
```suggestion
function test_oscap_info {
	version="$1"
	stdout="$(mktemp)"
	stderr="$(mktemp)"
	ds="$(mktemp)"
    cp scap-ds.xml $ds
    sed -i "s/X.X/${version}/g" $ds

	$OSCAP info $ds > $stdout 2> $stderr
	[ ! -s $stderr ]
	grep -q "Version: $version" $stdout
	rm $stdout
	rm $stderr
    rm $ds
}

SDS=$(find ../../../schemas/sds -maxdepth 1 -mindepth 1 -type d -printf '%f\n')
echo $SDS

for sds_version in $SDS
do
    test_oscap_info $sds_version
done
```

Reply:
@ggbecker Great idea! Please submit that as a separate PR.

Reply:
Sure, I'll create a new PR to propose this solution. Thanks